matrix-core/matrix.tdl

```diff
@@ -178,15 +178,15 @@ expressed-non-canonical := non-canonical & expressed-synsem.
 
 gap := expressed-non-canonical &
   [ LOCAL #local,
-    NON-LOCAL [ REL 0-dlist,
-		QUE 0-dlist,
-		SLASH 1-dlist &
+    NON-LOCAL [ REL 0-alist,
+		QUE 0-alist,
+		SLASH append-list &
```

Comment:
`SLASH append-list` is redundant because that is already the appropriate type for `SLASH`.  The old version was also specifying `1-dlist`, but here we can simply have `SLASH.LIST < #local >`.

I would recommend removing all of the redundant `SLASH append-list` constraints.  This is not a critical problem (after all, it evaluates to the same feature structure), and so this could also be done later, perhaps along with removing the 0-alist type.

Reply:
Thank you, I addressed this (and cleaned up the 0-alist usage in matrix.tdl; they still are present in some libraries for now).